akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -162,23 +165,16 @@ abstract class ActorMaterializer extends Materializer {
    */
   def isShutdown: Boolean
 
-  /**
-   * INTERNAL API: this might become public later
-   */
-  private[akka] def actorOf(context: MaterializationContext, props: Props): ActorRef
+  def materialize[Mat](_runnableGraph: Graph[ClosedShape, Mat],
+                       subflowFuser: GraphInterpreterShell ⇒ ActorRef): Mat
 
-  /**
-   * INTERNAL API
-   */
-  private[akka] def system: ActorSystem
+  def actorOf(context: MaterializationContext, props: Props): ActorRef
 
-  /**
-   * INTERNAL API
-   */
-  private[akka] def logger: LoggingAdapter
+  def system: ActorSystem
+
+  def logger: LoggingAdapter
```

Comment:
We have a ticket for logging in graph stage, but it will probably not be solved like this.
https://github.com/akka/akka/issues/18793

Can we keep this closed for now. You can create your own LoggingAdapter via the system, `akka.event.Logging(mat.system, classOf[Something])`